index.js

```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+const cors = require("cors");
+const app = express();
+const PROJECTS = require("./const/projects").PROJECTS;
+
+app.use(express.json());
+app.use(cors());
+app.use(express.static("public"));
+
+const PORT = process.env.PORT || 3001;
```

Comment:
Feel free to use https://www.npmjs.com/package/dotenv package and put .env file to the project
Good practice is to store all private constants (e.g. database creds, tokens, host, port, etc) in `.env` file, add it to `.gitignore` and provide `env.example` file for developers